File: db4on/reference/db4o-reference-chapters/src/com/db4odoc/f1/selpersist/TestCustomized.cs
/* Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com */
using System;

namespace com.db4odoc.f1.selpersist
{
	public class TestCusomized
	{
		[com.db4odoc.f1.selpersist.FieldTransient]
		string _transientField;
		string _persistentField;

		public TestCusomized(string transientField, string persistentField)
		{
			_transientField = transientField;
			_persistentField = persistentField;
		}

		public override String ToString() 
		{
			return "Customized test: persistent: " + _persistentField + ", transient: " + _transientField ;
		}
	}
}

File: db4on/reference/vb/db4o-reference-chapters/src/com/db4odoc/f1/selpersist/TestCustomized.vb
' Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com 
Imports System

Namespace com.db4odoc.f1.selpersist
    Public Class TestCusomized
        <com.db4odoc.f1.selpersist.FieldTransient()> _
        Dim _transientField As String
        Dim _persistentField As String

        Public Sub New(ByVal transientField As String, ByVal persistentField As String)
            _transientField = transientField
            _persistentField = persistentField
        End Sub

        Public Overrides Function ToString() As String
            Return "Customized test: persistent: " + _persistentField + ", transient: " + _transientField
        End Function
    End Class
End Namespace
